docs/deploy.rst

```diff
@@ -22,6 +22,7 @@ pairs, exodus-storage.yaml, run by the `AWS CLI`_ command
         Default: exodus
     - oai:
         The origin access identity ID associated with the environment.
+        `AWS CloudFront`_ origin access identities are created separately.
```

Comment:
It's not clear to me what this is trying to communicate to the reader.  Can it be spelled out in a little more detail?  Is it trying to say that, in order to find the value for this parameter, you should look at a cloudfront distribution and extract it from there?  Or trying to say that you should not share an oai between here and cloudfront?

Reply:
I guess this sentence isn't necessary as the parameter's existence implies that the origin access identity isn't created here, which is what I intended to convey.

Reply:
I'll rewrite this to be more helpful.